hatchways/team-honey-cruller
# Forgot password

### What this PR does (required):
- Allow users to reset their password if they forgot it

### Screenshots / Videos (required):
![Screen Shot 2021-08-04 at 2 38 25 PM](https://user-images.githubusercontent.com/72681019/128258701-e304e818-03cd-4880-8622-befed63ef0cd.png)

![Screen Shot 2021-08-04 at 2 38 40 PM](https://user-images.githubusercontent.com/72681019/128258730-26a72a37-2449-4b53-9034-3c032b33e9ac.png)

![Screen Shot 2021-08-04 at 2 38 05 PM](https://user-images.githubusercontent.com/72681019/128258680-95916a7f-757d-4b17-b8ab-1a9980fa665a.png)


### Any information needed to test this feature (required):
- Go to the login page and click forgot password
- Enter your email
- Go to your email spam folder and open an email from honey cruller
- Click on the reset link and type in new password information
- You should now be able to log in with new password
